### PR TITLE
Standardized integration test files

### PR DESCRIPTION
**Pull request by kunagpal**
@czardoz @shamasis 
The integration test file names have been standardized according to the following convention:

1. File names with multiple words must have `-` as a separator, not `_`. Under no circumstances do file names involve `camelCase`.
2. The type of the file must be designated with `postman_{{type}}` in the file name, where `type` is one of `collection`. `data`, `environment`, or `globals`. See examples below for clarity.
3. `.json` is a common file extension across all files, data files can have `.csv` as an extension as well.

For instance:
>
file-name.postman_collection.json
file-name.postman_environment.json
file-name.postman_globals.json
file-name.postman_data.json
file-name.postman_data.csv

Also, all collections in the *V1* format have been converted to *V2*.

### CODE REVIEW

File: test/integration/csv-comma.postman_data.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+nameWithComma,nameWithoutComma
+"Abhijit, Kane","Abhijit Kane"
```
Comment (czardoz) [inline test/integration/csv-comma.postman_data.csv]: Can we give some random data here? @abhijitkane

File: test/integration/e2.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+	"id": "426c4896-2d5b-80b8-da5d-ac205502f3c0",
+	"name": "testEnv",
+	"values": [
+		{
+			"key": "envKey",
+			"value": "abhijit3",
```
Comment (czardoz) [inline test/integration/e2.postman_environment.json]: Change to something else

File: test/integration/csv-comma.postman_data.csv (hunk 1 above)
Comment (czardoz) [inline test/integration/csv-comma.postman_data.csv]: Actually, no, we'll take it up later.

**Comment by czardoz** [comment]
@kunagpal Can you merge the latest `feature/v3` branch into yours? 
I can see the iteration data changes in this PR.

File: temp.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
+{
```
Comment (czardoz) [inline temp.json]: This should not be commited

File: npm/test-integration.js [hunk 4]
```diff
@@ -1,6 +1,90 @@
...
+require('shelljs/global');
+require('colors');
 
-// banner line
-echo('Running Integration Tests');
-echo('- nothing to run yet!');
+var fs = require('fs'),
+    _ = require('lodash'),
+    async = require('async'),
+    assert = require('assert'),
```
Comment (czardoz) [inline npm/test-integration.js]: Can you use `express.js`?

File: npm/test-integration.js [hunk 5]
```diff
@@ -1,6 +1,90 @@
...
+    newman = require('../index'),
+
+    CONCURRENCY_LIMIT = 20,
+    SPEC_SOURCE_DIR = './test/integration';
```
Comment (czardoz) [inline npm/test-integration.js]: should be `path.join(__dirname, .......)`

File: npm/test-integration.js (hunk 4 above)
Comment (kunagpal) [inline npm/test-integration.js]: Did you mean `expect.js`?

File: temp.json (hunk 3 above)
Comment (kunagpal) [inline temp.json]: Yes, this is meant to be a temporary reference. I'll remove it anyway.

**Comment by shamasis** [comment]
Great. Good to go from my end.

File: npm/test-integration.js (hunk 4 above)
Comment (czardoz) [inline npm/test-integration.js]: I did, sorry for the confusion

File: npm/test-integration.js [hunk 6]
```diff
@@ -1,6 +1,91 @@
...
+            }, next);
+        },
+        function (compiledTests, next) {
+            var integrationTests = compiledTests[0],
+                collections = Object.keys(integrationTests);
+
+            console.log(`Running sanity tests for ${collections.length} collections...`.yellow.bold);
+
+            // run tests using the consolidated test set as a guide
+            async.mapLimit(collections, CONCURRENCY_LIMIT, function (test, testCallback) {
```
Comment (czardoz) [inline npm/test-integration.js]: A console log with the collection name will help a lot with debugging :-)

**Comment by shamasis** [comment]
One more thing. Should we set reporter to CLI?

**Comment by shamasis** [comment]
Not so important since the failures are printed at end.

File: npm/test-integration.js (hunk 6 above)
Comment (kunagpal) [inline npm/test-integration.js]: Okay, I'll add it.

Comment (czardoz) [inline npm/test-integration.js]: Let's make them serial then, and get rid of the concurrancy. Having fast tests is not very useful if you can't figure out why they are failing.

**Pull request merged**